addons/binding/org.openhab.binding.konnected/.gitignore

```diff
@@ -0,0 +1,2 @@
+/output/
+/target/
```

Comment:
Can this .gitignore file be removed? 

All ignored files from the parent directories are inherited which includes:
https://github.com/openhab/openhab2-addons/blob/master/.gitignore

Reply:
yes this was a holdover from when i was initially just pushing this code to its own repository before i added it to the addons as a fork for the pull request

Reply:
There are still two new .gitignore added in this PR

Reply:
I keep deleting these and they keep coming back.